### ISSUE TITLE
[Ambient Occlusion] Ghostly afterimage while moving/interacting

### ISSUE THREAD

**Issue by BuckPrivate** [sp614x/optifine#6149]
### Description of Issue
While mining, moving, or interacting with the world in torchlight with Ambient Occlusion on, ghostly afterimages of items in hands (i.e., shield, pickaxe) remain for a few seconds. Turning Ambient Occlusion off resolves the issue.

https://user-images.githubusercontent.com/60932187/128776293-40348ace-7713-437d-bf18-3343aab91449.mp4

## Steps to Reproduce
1. Load up Minecraft with Sildur's Vibrant Shaders
2. Aim at torch
3. Wiggle
4. Observe ghostly pickaxe/sword/shield/whatever afterimages

## OptiFine Version
1.17.1 Optifine HD U G9 pre33

## Installation Method
Standalone

## Other Installed Mods
Sildur's Vibrant Shaders 1.29 Extreme w/ Volumetric Lighting

## F3 Debug Screenshot
![2021-08-09_16 20 29](https://user-images.githubusercontent.com/60932187/128776106-5b11b9c5-5810-4899-86c8-d168c9a16f4c.png)

## Prior Testing
- Does this happen in Vanilla without OptiFine?
Negative

**Comment by sp614x** [comment]
Does this happen without shaders?

**Comment by BuckPrivate** [comment]
No. It also doesn't happen in the Nether, or when torchlight is in an area with natural light. It's only when the light is only torchlight.
